classes/scons.oeclass

```diff
@@ -0,0 +1,28 @@
+CLASS_DEPENDS += "native:scons"
+
+inherit c
+
+EXTRA_OESCONS ?= ""
+
+OESCONS = " \
+    --prefix=${D}${prefix} \
+    TARGET_ARCH=${TARGET_ARCH} \
+    TARGET_OS=linux \
+"
+
+OESCONS:>cxx += "CXX=${HOST_CXX}"
+OESCONS:>c += "CC=${HOST_CC}"
```

Comment:
You need to manually add a space in front of both CXX and CC, as the :> operator does not, and the += only adds a space between the current value (which is fx. OESCONS:>c) and the append value, and not the parent value (ie. OESCONS).